config.md

```diff
@@ -340,6 +340,10 @@ For Linux-based systems, the `process` object supports the following process-spe
 
     * **`class`** (string, REQUIRED) specifies the I/O scheduling class. Possible values are `IOPRIO_CLASS_RT`, `IOPRIO_CLASS_BE`, and `IOPRIO_CLASS_IDLE`.
     * **`priority`** (int, REQUIRED) specifies the priority level within the class. The value should be an integer ranging from 0 (highest) to 7 (lowest).
+* **`cpuAffinity`** (object, OPTIONAL) specifies CPU affinity used to execute the process.
+    The following properties are available:
+    * **`initial`** (string, REQUIRED) is a list of CPUs the process will be run on initially, before the transition to container's cgroup.
```

Comment:
The string format needs to be explained (with an example)

Reply:
Thanks; updated.